tests/test_scalar_comprehensive.py

```
@@ -0,0 +1,495 @@
+        s71.insert_deriv('t', Scalar(2.))
+        s72 = s71.to_scalar(0, recursive=False)
+        self.assertEqual(len(s72.derivs), 0)
+
+        # Test as_index with masked parameter
+        s73 = Scalar([0, 1, 2, 3])
+        idx3 = s73.as_index(masked=99)
+        self.assertTrue(np.allclose(idx3, [0, 1, 2, 3]))
+
+        # Test as_index_and_mask with masked parameter
+        s74 = Scalar([0, 1, 2])
+        idx4, mask4 = s74.as_index_and_mask(masked=99)
```

Ruff (0.14.7)

Unpacked variable `mask4` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)